### ISSUE TITLE
Inifinity loop that hangs Postman

### ISSUE THREAD

**Issue by EloB** [postmanlabs/postman-app-support#923]
OS X
Chrome Version 40.0.2214.111 (64-bit)
Postman Version 0.8.4.12

To trigger the bug just include `<meta http-equiv="refresh" content="0" />` in the response.

Here is a node server that triggers the bug. Run this node server and make a `GET` request to `http://localhost:8081/` in Postman. Then you will see that Postman will inifinity refresh the Postman page and you can't close the tab or do anything.

```js
var http = require('http');

http.createServer(function(req, res) {
  res.end([
    '<!DOCTYPE html>',
    '<html>',
      '<head>',
        '<title>Infinity refresh bug</title>',
        '<meta http-equiv="refresh" content="0" />',
      '</head>',
      '<body></body>',
    '</html>'
  ].join('\n'));
}).listen(8081, function(err) {
  console.log(err ? 'Failed to start webserver' : 'Started webserver');
});
```

**Comment by abhijitkane** [comment]
This isn't a problem with the packaged app (https://chrome.google.com/webstore/detail/postman-rest-client-packa/fhbjgbiflinjbdggehcddcbncdddomop?hl=en).

We'll push a fix for v0.8.4.12 soon.

**Comment by EloB** [comment]
Good!
